docs/node-operators/docker-compose-service.md

```diff
@@ -0,0 +1,77 @@
+# Run as `docker-compose` services
+
+## Prerequesites
```

Comment:
```suggestion
## Prerequisites
```

Reply:
solve by signed commit below, thank you for reporting